h2o-core/src/main/java/water/parser/CsvParser.java

```diff
@@ -59,7 +58,8 @@
     boolean decimal = false;
     int fractionDigits = 0;
     int tokenStart = 0; // used for numeric token to backtrace if not successful
-    int colIdx = 0;
+    int columnCounter = 0;  // index into parsed columns only, exclude skipped columns.
```

Comment:
Please let's name this `parsedColumnCounter`. The difference between `colIdx` and `columnCounter` is not really obvious. Or just `parsedCols` or even `parsedCntr` is ok.

Reply:
Pavel, sooooooooo picky!  Done.